crates/holochain/src/conductor/interface/websocket.rs

```diff
@@ -65,22 +65,25 @@ pub fn spawn_admin_interface_tasks<A: InterfaceApi>(
 
     tm.add_conductor_task_ignored(&format!("admin interface, port {}", port), |_stop| {
         async move {
-            let num_connections = Arc::new(AtomicIsize::new(0));
+            let mut active_connections = Vec::new();
```

Comment:
I can't find anything in the Tokio docs about this being a problem. We're only holding a `JoinHandle`, the task is free to proceed and finish. I don't expect that to be a problematic amount of resources to be holding. The benefit of not having accounting problems seems more valuable to me.